### DISCUSSION TITLE
[feature] Read Start URL and Region via Environment Variables

### DISCUSSION THREAD

**Discussion by theurichde** [theurichde/go-aws-sso#17] [General]
**Describe the solution you'd like**
Read the start URL and region from environment variables

**Additional context**
I would like to read the needed variables / flags via environment variables, like

```shell
AWS_DEFAULT_SSO_START_URL="https://my-login.awsapps.com/start#/"
AWS_DEFAULT_SSO_REGION="eu-central-1"
```

The chain should be

1.) Saved Config
2.) Environment Variables
3.) CLI Flags

where the predecessor always gets overridden

**Comment by bfanyuk** [comment]
And also aws profile to update :)

**Comment by theurichde** [comment]
I moved this issue to discussion as it was stale for 5 months. 

When there is a dedicated need for this feature, I will implement it. But for now there is simply no need.
